aws-marketplace-install.md

```diff
@@ -1,13 +1,15 @@
 AWS Marketplace Install
 =======================
 
-This document provides the steps for installing the Kubecost Enterprise product from the AWS marketplace. [More info on different tiers.](https://kubecost.com/pricing)
+This document provides the steps for installing the Kubecost product from the AWS marketplace. [More info on different tiers.](https://kubecost.com/pricing)
 
 Please contact us at support@kubecost.com with any questions and we'd be happy to help!
 
 ## Step 1. Create an IAM policy
 
-[More info on how to create a new policy](https://docs.aws.amazon.com/IAM/latest/UserGuide/tutorial_managed-policies.html#step1-create-policy)
+To deploy Kubecost from AWS Marketplace, you need to assign IAM policy with approriate IAM permission to a Kubernetes (K8s) service account before starting the deployment. You can either use AWS managed policy `arn:aws:iam::aws:policy/AWSMarketplaceMeteringRegisterUsage` or creating your own IAM policy. You can learn [more info on how to create a new policy](https://docs.aws.amazon.com/IAM/latest/UserGuide/tutorial_managed-policies.html#step1-create-policy)
```

Comment:
nit: grammar. "you need to assign _an_  IAM policy"

Reply:
I will fix it in the next commit.